content/observe/hook.mdx

```diff
@@ -37,4 +37,4 @@ function Person() {
 }
 ```
 
-Despite of using hook in the middle of the component, it will re-render a whole component on change of the observable. If you want to micro-manage renders, feel free to use [`<Observer />`](/observer-component) or `useForceUpdate` option (for advanced users).
\ No newline at end of file
+Despite using a `useObservable` hook in the middle of the component, it will re-render a whole component on change of the observable. If you want to micro-manage renders, feel free to use [`<Observer />`](/observer-component) or `useForceUpdate` option (for advanced users).

```

Comment:
```suggestion
Despite using a `useObserver` hook in the middle of the component, it will re-render a whole component on change of the observable. If you want to micro-manage renders, feel free to use [`<Observer />`](/observer-component) or `useForceUpdate` option (for advanced users).
```